src/scripts/assume_role_with_web_identity.sh

```diff
@@ -37,14 +37,17 @@ $(aws sts assume-role-with-web-identity \
 --output text)
 EOF
 
+temp_file="/tmp/${AWS_CLI_STR_PROFILE_NAME}.keys"
+touch "$temp_file"
+
 if [ -z "${AWS_ACCESS_KEY_ID}" ] || [ -z "${AWS_SECRET_ACCESS_KEY}" ] || [ -z "${AWS_SESSION_TOKEN}" ]; then
     echo "Failed to assume role";
     exit 1
 else 
     {
-        echo "export AWS_ACCESS_KEY_ID=\"${AWS_ACCESS_KEY_ID}\""
-        echo "export AWS_SECRET_ACCESS_KEY=\"${AWS_SECRET_ACCESS_KEY}\""
-        echo "export AWS_SESSION_TOKEN=\"${AWS_SESSION_TOKEN}\""
-    } >>"$BASH_ENV"
+        echo "export AWS_CLI_STR_ACCESS_KEY_ID=\"${AWS_ACCESS_KEY_ID}\""
+        echo "export AWS_CLI_STR_SECRET_ACCESS_KEY=\"${AWS_SECRET_ACCESS_KEY}\""
+        echo "export AWS_CLI_STR_SESSION_TOKEN=\"${AWS_SESSION_TOKEN}\""
```

Comment:
This is a breaking change for the orb. 

Most customers likely relied on the existence of `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN`. They are standard `env` variables that AWS clients automatically detect. They are now gone. 

Please roll back this PR on the 4.x branch.

Reply:
Agree

Reply:
Yup, this broke our CI build in a very confusing way.